pkg/scheduler/algorithm/priorities/resource_limits_test.go

```diff
@@ -140,7 +140,12 @@ func TestResourceLimistPriority(t *testing.T) {
 	for _, test := range tests {
 		t.Run(test.name, func(t *testing.T) {
 			nodeNameToInfo := schedulernodeinfo.CreateNodeNameToInfoMap(nil, test.nodes)
-			list, err := priorityFunction(ResourceLimitsPriorityMap, nil, nil)(test.pod, nodeNameToInfo, test.nodes)
+
+			metadata := &priorityMetadata{
+				podLimits: getResourceLimits(test.pod),
+			}
+
+			list, err := priorityFunction(ResourceLimitsPriorityMap, nil, metadata)(test.pod, nodeNameToInfo, test.nodes)
```

Comment:
You can add a `foo` loop to iterate `hasMeta` = true/false, then compose metadata or put it as `nil`.

Otherwise, the test case just verifies `hasMeta` = true.

Reply:
That's a good catch, it fixes a bug in [1]

Reply:
it may haves the same problem with ResourceAoolicationPriority, I'll open another PR against it.